src/libstd/net/ip.rs

```diff
@@ -357,47 +357,38 @@ impl Ipv4Addr {
     /// # Examples
     ///
     /// ```
-    /// #![feature(ip_constructors)]
     /// use std::net::Ipv4Addr;
     ///
     /// let addr = Ipv4Addr::LOCALHOST;
     /// assert_eq!(addr, Ipv4Addr::new(127, 0, 0, 1));
     /// ```
-    #[unstable(feature = "ip_constructors",
-               reason = "requires greater scrutiny before stabilization",
-               issue = "44582")]
+    #[stable(feature = "ip_constructors", since = "1.29.0")]
```

Comment:
The version should be 1.30.0

Reply:
fixed. Is there an easy way to see what the current nightly version is?

Reply:
@tmccombs http://play.rust-lang.org/ will show the nightly version in use if you open the channel dropdown menu. But also, it's basically always current stable +2 versions.